include/tbl/vsl_tags.h

```diff
@@ -138,15 +138,17 @@ SLTM(SessClose, 0, "Client connection closed",
 SLTM(BackendOpen, 0, "Backend connection opened",
 	"Logged when a new backend connection is opened.\n\n"
 	"The format is::\n\n"
-	"\t%d %s %s %s %s %s %s\n"
-	"\t|  |  |  |  |  |  |\n"
-	"\t|  |  |  |  |  |  +- \"connect\" or \"reuse\"\n"
-	"\t|  |  |  |  |  +---- Local port\n"
-	"\t|  |  |  |  +------- Local address\n"
-	"\t|  |  |  +---------- Remote port\n"
-	"\t|  |  +------------- Remote address\n"
-	"\t|  +---------------- Backend display name\n"
-	"\t+------------------- Connection file descriptor\n"
+	"\t%d %s %s %s %s %s %s [ %.6f %ld ]\n"
+	"\t|  |  |  |  |  |  |    |    |\n"
+	"\t|  |  |  |  |  |  |    |    +- Connection uses\n"
+	"\t|  |  |  |  |  |  |    +------ Connection age\n"
```

Comment:
Apologies for flip-flopping on this one but looking at the reused field it should probably say "Connection reuses" here.

Reply:
Thanks, addressed.